payments/base/oem-extractor/src/main/java/cm/aptoide/pt/oem_extractor/extractors/BaseOemIdExtractor.kt

```diff
@@ -0,0 +1,13 @@
+package cm.aptoide.pt.oem_extractor.extractors
+
+import android.content.Context
+
+internal abstract class BaseOemIdExtractor(
+  private val context: Context
+) : IExtractOemId {
+  protected fun getPackageName(packageName: String): String {
```

Comment:
Get a package name by the package name? 🥴 
What does it do?
Looks like a candidate for a local extension function...